### ISSUE TITLE
Tracker for preparing textbox2.0 Movie Dialogue dataset

### ISSUE THREAD

**Issue by bennmann** [LAION-AI/Open-Assistant#1174]
https://huggingface.co/datasets/RUCAIBox/Open-Dialogue/blob/main/md.tgz

Task to onvert to something like:

Write me a {dialog/script/play/etc) between {characters1...N} about {whatever the subject is of the movie}"

Here's a {dialog/script/play/etc) between {characters1...N} about {whatever the subject is of the movie}:

character 1: dialog
character 2: dialog
etc.

Vary so that we can have follow ups questions. "What would character 2 say if {...}"

Basically we are try to create a dialog for someone who might want to create a movie script :)

Related to:
https://github.com/LAION-AI/Open-Assistant/issues/1163](https://github.com/LAION-AI/Open-Assistant/issues/1163)

**Comment by bennmann** [comment]
It would be good to have a team members thoughts on the data format based on SOTA data preparation described here, but for the movie dialogue above:

https://github.com/microsoft/GODEL/tree/main/data/grounded

**Comment by ontocord** [comment]
Can we consolidate with this issue: https://github.com/LAION-AI/Open-Assistant/issues/1163

I can assign to you @bennmann too

**Comment by bennmann** [comment]
Did a quick zcat and less on a random sample of this data and it will require intensive cleaning and many steps to make good enough for an assistant... I will investigate other external datasets in the meantime
